src/schema/v2/me/loadBatchPriceInsights.ts

```diff
@@ -0,0 +1,40 @@
+import gql from "lib/gql"
+import { extractNodes } from "lib/helpers"
+
+const MAX_MARKET_PRICE_INSIGHTS = 50
+
+export const loadBatchPriceInsights = async (
```

Comment:
I wonder if this file should live under `/me`. The rest of the files are about fields. Maybe we can move it to `/lib` 🤔